subprojects/core/src/main/java/org/gradle/caching/configuration/internal/DefaultBuildCacheConfiguration.java

```diff
@@ -71,6 +72,7 @@ public void setLocal(DirectoryBuildCache local) {
     }
 
     @Override
+    @Deprecated
```

Comment:
Why do we need these repeated? I don't see any warnings in IntelliJ for this class currently.

Reply:
Not all warnings and errors are aligned with IDEA yet. Something we also have to work on. But that's a general problem. When you remove this and compile with Gradle, it'll fail.

Reply:
We already talked about this topic in the team and I think @bmuschko (and maybe @ljacomet?) did some investigation. I'll see that we follow up on this.

Reply:
👍 Okay.

-